ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantSessionManager.scala

```diff
@@ -3,26 +3,62 @@
 
 package com.daml.ledger.api.testtool.infrastructure.participant
 
+import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantSessionManager._
+import io.grpc.ManagedChannel
 import io.grpc.netty.{NegotiationType, NettyChannelBuilder}
+import io.netty.channel.EventLoopGroup
 import io.netty.channel.nio.NioEventLoopGroup
 import io.netty.channel.socket.nio.NioSocketChannel
 import io.netty.util.concurrent.DefaultThreadFactory
 import org.slf4j.LoggerFactory
 
-import scala.collection.concurrent.TrieMap
+import scala.collection.immutable
+import scala.concurrent.duration.SECONDS
 import scala.concurrent.{ExecutionContext, Future}
 
-private[infrastructure] final class ParticipantSessionManager {
+private[infrastructure] final class ParticipantSessionManager(
+    sessions: immutable.Map[ParticipantSessionConfiguration, SessionParts],
```

Comment:
`Map`s are immutable by default in Scala, no need to specify it.

Reply:
Yup, fixed.